packages/core/stories/patterns/wizard/wizard.stories.tsx

```diff
@@ -137,8 +398,7 @@ const ContentOverflow = ({
   const divRef = useRef<HTMLDivElement>(null);
 
   const handleScroll = () => {
-    const targetWindow = window;
-    targetWindow?.requestAnimationFrame(() => {
+    window?.requestAnimationFrame(() => {
```

Comment:
I guess this was supposed to have desktop support? If we're not added it can we drop `window?.` ?